packages/gateway/package.json

```diff
@@ -7,7 +7,7 @@
     "postinstall": "if [ \"$NODE_ENV\" != \"production\" ]; then npx crlf --set=LF ../../node_modules/graphql-schema-typescript/**/* || npx crlf --set=LF ./node_modules/graphql-schema-typescript/**/* || true; fi",
```

Comment:
## Gateway

Notice that gateway now is responsible for turning those `role: 'FIELD_AGENT'` labels into the expanded form of 

```ts
export type UserRole = {
  __typename?: 'UserRole'
  id: Scalars['ID']
  label: I18nMessage
  scopes: Array<Scalars['String']>
  systemRole: SystemRoleType
}
```